ember-server/shared/src/main/scala/org/http4s/ember/server/EmberServerBuilder.scala

```diff
@@ -176,6 +179,23 @@ final class EmberServerBuilder[F[_]: Async: Network] private (
   ): EmberServerBuilder[F] =
     copy(additionalSocketOptions = additionalSocketOptions)
 
+  /** An error handler which will run in cases where the server is unable to
+    * parse the "start-line" (http 2 name) or "request-line" (http 1.1
+    * name). This is the first line of the request, e.g. "GET / HTTP/1.1".
+    *
+    * In this case, RFC 9112 (HTTP 2) says a 400 should be returned.
+    *
+    * This handler allows for configuring the behavior. The default as of
+    * 0.23.19 is to return a 400.
+    *
+    * @see [[https://www.rfc-editor.org/rfc/rfc9112#section-2.2-9 RFC 9112]
```

Comment:
```suggestion
    * @see [[https://www.rfc-editor.org/rfc/rfc9112#section-2.2-9 RFC 9112]]
```

Fixes the scaladoc formatting

Reply:
Thanks @valencik. Updated.